test/index.spec.js

```diff
@@ -44,13 +46,14 @@ describe('webpack-rxjs-externals', () => {
         const expectedOutput = fs.readFileSync(path.join(__dirname, fixturesDir, fixture, 'expected', 'index.js')).toString();
 
         const compiler = webpack(config);
-        compiler.outputFileSystem = new MemoryFileSystem();
+        const memoryFileSystem = createFsFromVolume(new Volume());
+        compiler.outputFileSystem = memoryFileSystem;
 
         compiler.run((err, stats) => {
           if (err) {
             reject(err);
           }
-          const actualOutput = stats.compilation.assets['index.js'].source();
+          const actualOutput = memoryFileSystem.readFileSync('/index.js').toString();
```

Comment:
This was replaced b/c compilation assets doesn't have sources in webpack@5 https://webpack.js.org/blog/2020-10-10-webpack-5-release/#sizeonlysource-after-emit
And overall reading fs output looks better.